### DISCUSSION TITLE
Is it possible to use magic-move with twoslash?

### DISCUSSION THREAD

**Discussion by hpohlmeyer** [slidevjs/slidev#1904] [Q&A] [answered]
I am currently trying to combine twoslash with magic-move, but cannot get it working. Is combining both supported, and I am using the wrong syntax, or is it unsupported?

	````md magic-move
	```ts twoslash
	let nope: never;
	nope = "string";
	```
	```ts twoslash
	let nope: never;
	nope = boolean;
	```
	````

**Comment by kermanx** [comment, marked as answer]
I don't think this is possible yet. We may have to wait for https://github.com/shikijs/shiki-magic-move/issues/19.

**Reply by hpohlmeyer** [reply]
Thanks, I was expecting that. I guess it is pretty tricky to let them work together properly.
